partition/gpt/table.go

```diff
@@ -347,15 +347,9 @@ func readPartitionArrayBytes(b []byte, entrySize, logicalSectorSize, physicalSec
 	return parts, nil
 }
 
-// tableFromBytes read a partition table from a byte slice
-func tableFromBytes(b []byte, logicalBlockSize, physicalBlockSize int) (*Table, error) {
-	// minimum size - gpt entries + header + LBA0 for (protective) MBR
-	if len(b) < logicalBlockSize*2 {
-		return nil, fmt.Errorf("data for partition was %d bytes instead of expected minimum %d", len(b), logicalBlockSize*2)
-	}
-
-	// GPT starts at LBA1
-	gpt := b[logicalBlockSize:]
+// readGptHeader reads the GPT header from the given byte slice
+func readGptHeader(b []byte) (*Table, error) {
```

Comment:
I suspect lint will complain that this should be `readGPTHeader()`

Reply:
Fixed.